src/components/molecules/results-list/results-list-item/results-list-item.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { useStateValue } from '../../../../store/store';
+import { testIds } from '../../../../constants';
+
+const ResultsListItem = ({ nciId, locationInfo, summary, title }) => {
+	const [{ resultsItemTitleLink }] = useStateValue();
+	return (
+		<li className="ct-list-item">
+			<div className="title-and-desc title desc container">
```

Comment:
ooof.  we can probably lose this wrapper div so that it's simply:
```
<li>
  <a />
  <div class="body" />
</li>
```

Reply:
Done!